README.md

```diff
@@ -83,7 +83,7 @@ If you want to do some more things in a exceptions app, you can also write your
 $ rails g rambulance:exceptions_app
 ```
 
-It will generate your own custom exceptions app. You can use whatever techniques you want to use in controllers like `before_filter` and `flash[:notice] = "message..."` since it's a grandchild of `ActionController::Base`!
+It will generate your own custom exceptions app. You can use most techniques you want to use in controllers like `before_filter`, `flash[:notice] = "message..."` and rendering views since it's a grandchild of `ActionController::Base`! However there are still some restrictions, e.g. setting a flash notice works when rendering directly but not when redirecting because the ActionDispatch::Flash middleware is never hit.
```

Comment:
```suggestion
It will generate your own custom exceptions app. You can use most techniques you want to use in controllers like `before_filter` and rendering views since it's a grandchild of `ActionController::Base`! However there are still some restrictions, e.g. setting a flash notice works when rendering directly but not when redirecting because the ActionDispatch::Flash middleware is never hit.
```